website/src/components/FAQSection/index.tsx

```diff
@@ -42,8 +42,8 @@ const accordionData: AccordionItem[] = [
         prior to the BUSL. This corresponds to a few commits after 1.14.8,
         but prior to 1.14.9 being cut.
 
-        From here forward, we'll be adhering to our
-        <a href="/docs/policies/migration/#proposal">compatibility policy</a>
+        From here forward, we'll be adhering to our{" "}
+        <a href="/docs/policies/migration/#proposal">{" "}compatibility policy</a>
```

Comment:
```suggestion
        From here forward, we'll be adhering to our{" "}
        <a href="/docs/policies/migration/#proposal">compatibility policy</a>{" "}
```

Doesn't the current version mean we'll end up with two spaces in front and none behind?

Reply:
It does, you're right.